### DISCUSSION TITLE
Failed to compile. Global CSS cannot be imported from within node_modules.

### DISCUSSION THREAD

**Discussion by jabed-web-dev** [tailwindlabs/tailwindcss#8999] [Help]
I'd run the project with 'yarn dev' but it's an error. How can I solve the problem?

![image](https://user-images.githubusercontent.com/59810760/182012865-a97a1c94-0071-422b-979a-2ad96afbf76d.png)

**Comment by wongjn** [comment]
From the error message, it looks like it's an incompatibility with the `monaco-editor` package.

[From the webpage directed by the error message](https://nextjs.org/docs/messages/css-npm):
> One of your dependencies (`node_modules`) imports a CSS file.
>
> […]
>
> Compiled dependencies do not have references to CSS files, or any other files that require bundler-specific integrations.
> 
> The dependency should also provide instructions about what CSS needs to be imported by you, in your application.

Looking at the README from [`monaco-editor`](https://github.com/microsoft/monaco-editor), it has several types of bundles one can use:

> You will get:
> - inside `esm`: ESM version of the editor (compatible with e.g. webpack)
> - inside `dev`: AMD bundled, not minified
> - inside `min`: AMD bundled, and minified
> […]
> It is recommended to develop against the dev version, and in production to use the min version.

It seems like you are importing the ESM version, but as stated above, this is for webpack compatibility and this sounds like it falls within the "bundler-specific integrations" and thus would not be compatible.

Try following the monaco editor's recommendation and use `dev` instead for development and `min` in production.
